code/__DEFINES/layers.dm

```diff
@@ -100,7 +100,10 @@
 
 #define BLACKNESS_PLANE 0 //To keep from conflicts with SEE_BLACKNESS internals
 
-
+/// The map used to mask the plane.
```

Comment:
what's "the plane"?

Reply:
I think I crossed map and plane in the doc. I'll fix it.